tmd/view/view.py

```diff
@@ -499,7 +499,7 @@ def population(
         return None, "No such plane found! Please select one of: xy, xz, yx, yz, zx, zy."
 
     # Initialization of matplotlib figure and axes.
-    fig, ax = _cm.get_figure(new_fig=new_fig, subplot=subplot)
+    fig, ax = cm.get_figure(new_fig=new_fig, subplot=subplot)
 
     kwargs["new_fig"] = False
     kwargs["subplot"] = subplot
```

Comment:
They are not default, because they need to change after the first pass. So you create a figure initially, but then each subsequent object (neuron) needs to be in the same figure. That's because we create a population of cells.

Reply:
Ah yeah ok I see